src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_helpers.cc

```diff
@@ -22,4 +28,45 @@ RCAppExtensionPtr RCHelpers::GetRCExtension(
           extension_interface);
   return extension;
 }
+
+smart_objects::SmartObjectSPtr RCHelpers::CreateUnsubscribeRequestToHMI(
+    const std::string& module_type, const uint32_t correlation_id) {
+  using namespace smart_objects;
+  namespace commands = application_manager::commands;
+  namespace am_strings = application_manager::strings;
+
+  SmartObjectSPtr message = utils::MakeShared<SmartObject>(SmartType_Map);
+  SmartObject& ref = *message;
+
+  ref[am_strings::params][am_strings::message_type] =
+      static_cast<int>(application_manager::kRequest);
+  ref[am_strings::params][am_strings::protocol_version] =
+      commands::CommandImpl::protocol_version_;
+  ref[am_strings::params][am_strings::protocol_type] =
+      commands::CommandImpl::hmi_protocol_type_;
+  ref[am_strings::params][am_strings::correlation_id] = correlation_id;
+  ref[am_strings::params][am_strings::function_id] =
+      hmi_apis::FunctionID::RC_GetInteriorVehicleData;
+  ref[am_strings::msg_params][message_params::kSubscribe] = false;
+  ref[am_strings::msg_params][message_params::kModuleType] = module_type;
+  return message;
+}
+
+std::vector<application_manager::ApplicationSharedPtr>
+RCHelpers::AppsSubscribedTo(application_manager::ApplicationManager& app_mngr,
+                            const std::string& module_type) {
+  std::vector<application_manager::ApplicationSharedPtr> result;
+  auto rc_apps = RCRPCPlugin::GetRCApplications(app_mngr);
+  for (auto& app : rc_apps) {
+    auto rc_ext = RCHelpers::GetRCExtension(*app);
+    DCHECK_OR_RETURN(rc_ext, result);
+    if (rc_ext->IsSubscibedToInteriorVehicleData(module_type)) {
+      result.push_back(app);
+    }
+  }
+  LOG4CXX_TRACE(logger_,
+                "AppsSubscribedTo " << module_type << " count "
```

Comment:
@LuxoftAKutsan Divide `AppsSubscribedTo` into three separate words

Reply:
@AByzhynar removed this trace